app/controllers/report_controller/reports/editor.rb

```diff
@@ -628,6 +628,10 @@ def gfv_performance
     end
   end
 
+  def chargeback_model?(model)
+    %w(ChargebackContainerProject ChargebackVm).include?(model)
```

Comment:
@zeari as you know we have now `db_is_chargeback?(db)`
https://github.com/ManageIQ/manageiq/pull/9238/files#diff-06b291c88bf5d62e6e8089364aa462d0L185